### PR TITLE
chore: move washboard to its own directory

### PR DESCRIPTION
**Pull request by connorsmith256**
## Feature or Problem
This is the minimal amount of work to split the `washboard` directory out of `wash` into its own top-level directory

The washboard doesn't have any CI yet

@lachieh should I commit changes to `dist/`?

### Manual Verification
- built the washboard: `cd washboard && make build-ui && cd -`
- built wash: `cd wash && cargo build`
- ran wash: `./target/debug/wash up --nats-websocket-port 4001`
- ran the washboard: `./target/debug/wash ui --experimental`

### CODE REVIEW

**Comment by lachieh** [comment]
Yes please, until we have a release process

**Review by rvolosatovs — APPROVED** [review]

**Comment by connorsmith256** [comment]
On second thought, I'll leave this in draft until we've archived the wash repo. I don't want to make it more difficult to cherry-pick changes from there

File: washboard-ui/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+##@ Building
+
+build-ui: ## Build the UI from source
+	@$(NPM) install
+	@$(NPM) run build
+
+##@ Running
+
+run-ui: ## Run UI from source
+	@$(NPM) instal
```
Comment (lachieh) [inline washboard-ui/Makefile:21]: I think you dropped this `l`

**Review by lachieh — CHANGES_REQUESTED** [review]

**Review by rvolosatovs — APPROVED** [review]
I'm very surprised that using the relative path to parent directory for the washboard worked here

File: washboard-ui/Makefile (hunk 1 above)
Comment (connorsmith256) [reply washboard-ui/Makefile]: Oops! Nice catch 🙏

**Review by lachieh — APPROVED** [review]

**Review by brooksmtownsend — APPROVED** [review]

**Comment by rvolosatovs** [comment]
Looks like you forgot to update the `paths` triggers for wash CI, which builds UI

**Pull request merged**